global_settings/wagtail_hooks.py

```diff
@@ -36,12 +36,20 @@ def register_strikethrough_feature(features):
 
 @hooks.register('after_edit_page')
```

Comment:
are all API routes considered pages?  said another way, can you edit something that changes an API endpoint but that doesn't trigger this hook?

Reply:
all pages and books are considered pages
good question though, let me think about this a bit more before merging - there might be a few instances where this is true and will not work as expected